docs/api/annotations.md

```diff
@@ -0,0 +1,394 @@
+`comment` | `string` | **Required** An optional extra value.
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+   "updated":"2019-04-14T07:55:16Z",
+   "created":"2018-06-07T13:50:44Z",
+   "id":5646929551687680,
+   "doneBase":null,
+   "doneFull":null
+   "assignment":4873731453222912,
+   "recording":{
+      "recording":"5715999101812736",
+      "organisation":"org1",
+      "challenge":"challenge1",
+      "digest":null,
+      "tenant":"demo"
+   },
+   "audioUrl":"https://api.itslanguage.io/download/3357a2f8ada1446bae8d97f591169de9",
+   "fragments":[
+      {
+         "start":0.59,
+         "end":1.07,
+         "labels":[
+            {
+               "layer":"Words",
+               "label":"zon",
+               "comment":""
+            },
+            {
+               "layer":"OutOfPrompt",
+               "label":"InsertionUtterance",
+               "comment":"zo"
+            }
+         ]
+      },
+      {
+         "start":9.81,
+         "end":10.93,
+         "labels":[
+            {
+               "layer":"Words",
+               "label":"weer.",
+               "comment":""
+            }
+         ]
+     }
+   ]
+}
+```
+
+## Update an annotation
+
+### URL
+
+```http
+PUT /annotations/:annotation HTTP/1.1
+```
+
+* `annotation` - **Required** The annotation identifier.
+
+Update one or more properties of an existing annotation.
+
+### Request parameters
+
+These are exactly as specified in [Create an annotation](#create-an-annotation).
+
+### Request
+
+The request is exactly as specified in [Create an annotation](#create-an-annotation).
+
+### Response
+
+The response is exactly as specified in [Create an annotation](#create-an-annotation).
+
+
+# Reference Annotations
+
+A reference annotation is an annotation for a specific audio recording, usually generated through
+forced alignment (matches words in the prompt of the challenge to the audio recording).
+That leads to annotations on the fragments where a relation between prompt and audio was found
+(the alignment). No error or out of prompt classes are assigned in an automated fashion. This is
+the starting point to create human annotations.
+
+## Get a single reference annotation
+
+When a reference annotation has been added for a specific recording at an earlier time, this is how
+to retrieve it. The URL includes the full namespace to the recording.
+
+### URL
+
+```http
+GET /organisations/:organisation/challenges/speech/:challenge/recordings/:recording/reference HTTP/1.1
+```
+### Request parameters
+
+These are exactly as specified in [Get a single annotation](#get-a-single-annotation).
+
+### Request
+
+The request is exactly as specified in [Get a single annotation](#get-a-single-annotation).
+
+### Response
+
+The response is exactly as specified in  [Get a single annotation](#get-a-single-annotation).
+
+
+## Update a reference annotation
```

Comment:
Perhaps it is good to mention that one does not simply has to create an reference annotation. It is there on the recording. That is why we do not describe a create but an update (PUT vs POST).

Reply:
You say it's there on the recording, but that sentence can easily be misunderstood as there is a reference annotation automatically generated. It's not. the field is there, the value is null/None.
I explained more reasoning for it. But carefully reading it, I'm not 100% sure about the reasons for not adding it as a field in the recording PUT. I suspect it's because of the security decorator. It'd require impersonation otherwise.

Reply:
I'll merge it now. We can debate history another time ;-)

Reply:
No need to, I see your latest amendment, and that is kinda what I meant, so this is fine enough now!

Thanks!